Tests/Shared.Specs/CollectionAssertionSpecs.cs

```diff
@@ -2337,6 +2336,231 @@ public void When_testing_collections_not_to_be_equivalent_against_same_collectio
 
         #endregion
 
+        #region Contain Equivalent Of
+
+        [Fact]
+        public void When_collection_contains_object_equal_to_another_it_should_succeed()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Arrange
+            //-----------------------------------------------------------------------------------------------------------
+            int item = 2;
+            IEnumerable collection = new[] { 1, item, 3 };
+
+            //-----------------------------------------------------------------------------------------------------------
+            // Act / Assert
+            //-----------------------------------------------------------------------------------------------------------
+            collection.Should().ContainEquivalentOf(item);
+        }
+
+        [Fact]
+        public void When_collection_contains_object_equivalent_to_another_it_should_succeed()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Arrange
+            //-----------------------------------------------------------------------------------------------------------
+            IEnumerable collection = new[] { 1, 2, 3 };
+            int item = 2;
+
+            //-----------------------------------------------------------------------------------------------------------
+            // Act / Assert
+            //-----------------------------------------------------------------------------------------------------------
+            collection.Should().ContainEquivalentOf(item);
+        }
+
+        [Fact]
+        public void When_character_collection_does_contain_equivalent_it_should_succeed()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Arrange
+            //-----------------------------------------------------------------------------------------------------------
+            char[] collection = ("abc123ab").ToCharArray();
+            char item = 'c';
```

Comment:
❓ Shouldn't this be testing against capital `'C'`?

Reply:
Why? With capital `'C'` it would fail because it is not in the char array.
`'C'.Should().BeEquivalentTo('c');` would fail too.

This only works with StringAssertions: `"C".Should().BeEquivalentTo("c");`
But StringAssertions will not be used here regardless if you use it on strings or not.
If StringAssertions should be used with string collection it would need to be implemented in the `StringCollectionAssertions` too. Right now it is only implemented in the `CollectionAssertions`.

I added the `When_string_collection_does_contain_same_string_with_other_case_it_should_throw` test to be clear about that.

Reply:
You're right. 
I confused it with the semantics `ContainEquivalentOf` from `StringAssertions`.